src/features/multipleChoiceAnswer/MultipleChoiceAnswer.tsx

```diff
@@ -0,0 +1,86 @@
+import { css } from '@emotion/react';
+
+import colors from '~/styles/color';
+import { BODY_2_BOLD } from '~/styles/typo';
+
+const MultipleChoiceAnswer = ({ variant = 'highlighted', totalCount = 3, answeredCount = 1 }) => {
+  return (
+    <div css={ContainerCss(variant)}>
+      <div css={PercentageBar(variant, totalCount, answeredCount)} />
+      <div css={TextContainerCss(variant)}>
+        <div css={[TextCss(variant)]}>UX가 좋습니다 졸려일무 무리무리무리루</div>
+        {/* TODO 아이콘 변경 예정 */}
+        <div css={[BODY_2_BOLD, CountTextCss(variant)]}>{answeredCount}명</div>
+      </div>
+    </div>
+  );
+};
+
+export default MultipleChoiceAnswer;
+
+const textStyles: Record<string, Record<string, string>> = {
+  default: {
+    fontWeight: '400',
+    fontSize: '1rem',
+    lineHeight: '150%',
+    letterSpacing: '-0.3px',
+  },
+  highlighted: {
+    fontWeight: '500',
+    fontSize: '1rem',
+    lineHeight: '150%',
+    letterSpacing: '-0.3px',
+  },
+};
+
+const ContainerCss = (variant: string) => css`
```

Comment:
```suggestion
const containerCss = (variant: string) => css`
```

문제는 없지만, `css` 들은 소문자로 시작하는게 자연스러워 보여요!

Reply:
아핫 그렇군요! 변경하겠습니다~!